compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -1143,6 +1293,20 @@ void mlir::concretelang::python::populateCompilerAPISubmodule(
              return pybind11::bytes(
                  clientParametersSerialize(clientParameters));
            })
+      .def("input_keyid_at",
```

Comment:
Should we return encryption parameters, as a pybing object and then have property on it, may be cleaner?

Reply:
i.e. have a full featured introspection API of client parameters.

Reply:
it might be too much code (given that the parameters have all this optional fields) that is unnecessary for now

Reply:
ok for this PR but I think we should take the opportunity to have a proper API for introspection, let open an issue